src/components/nodes/userTask/userTask.vue

```diff
@@ -1,22 +1,3 @@
-<template>
-  <crown-config
-    :highlighted="highlighted"
-    :paper="paper"
-    :graph="graph"
-    :shape="shape"
-    :node="node"
-    :nodeRegistry="nodeRegistry"
-    :moddle="moddle"
-    :collaboration="collaboration"
-    :process-node="processNode"
-    :plane-elements="planeElements"
-    :is-rendering="isRendering"
-    :dropdownData="dropdownData"
-    :boundary-event-dropdown-data="boundaryEventDropdownData"
-    v-on="$listeners"
-  />
-</template>
```

Comment:
This was unnecessary duplication since the template is declared on the parent component.